charts/spire/charts/spiffe-oidc-discovery-provider/files/test/jwt-decode.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bin/env
```

Comment:
This is an invalid shell.

Reply:
Im guessing busybox doesn't even read this, tests passed locally. Fixed